dubbo-admin-backend/src/main/java/org/apache/dubbo/admin/controller/AccessesController.java

```diff
@@ -95,6 +96,7 @@ public void deleteAccess(@PathVariable String id, @PathVariable String env) {
     }
 
     @RequestMapping(method = RequestMethod.POST)
+    @ResponseStatus(HttpStatus.CREATED)
     public void createAccess(@RequestBody AccessDTO accessDTO, @PathVariable String env) {
         if (StringUtils.isBlank(accessDTO.getService())) {
             throw new IllegalArgumentException("Service is required.");
```

Comment:
Can you replace the IllegalArgumentException with the corresponding exceptions.

Reply:
OK